runtime/reset.go

```diff
@@ -33,7 +33,7 @@ func (d *Default) reset(cluster *v1.Cluster) error {
 	if err != nil {
 		logger.Error("failed to clean masters %v", err)
 	}
-	err = utils.CleanFiles(common.GetClusterWorkDir(cluster.Name), common.DefaultClusterBaseDir(cluster.Name), common.DefaultKubeConfigDir())
+	err = utils.CleanFiles(common.GetClusterWorkDir(cluster.Name), common.DefaultKubeConfigDir())
```

Comment:
So how to clean the cluster basedir(/var/lib/sealer/data/my-cluster)

Reply:
this function will do this：
CleanFS: func(applier *DefaultApplier) error {
		return applier.FileSystem.Clean(applier.ClusterDesired)
	}